plugin/src/main/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfigurator.java

```diff
@@ -17,27 +21,45 @@
  */
 @Extension
 @Restricted(NoExternalUse.class)
-public class HudsonPrivateSecurityRealmConfigurator extends DataBoundConfigurator {
+public class HudsonPrivateSecurityRealmConfigurator extends DataBoundConfigurator<HudsonPrivateSecurityRealm> {
+    private static final Logger logger = LoggerFactory.getLogger(HudsonPrivateSecurityRealmConfigurator.class);
+    /// matches HudsonPrivateSecurityRealm.JBCRYPT_HEADER, requires Jenkins >= 164.1
+    private static final String HASHED_PASSWORD_PREFIX = "#jbcrypt:";
```

Comment:
https://github.com/jenkinsci/jenkins/blob/bf333b4637da11a9aa6e0e5e5bef5361bda2e35a/core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java#L875 should have been public but alas :cry:

Reply:
Indeed.